lib/features/home_page.dart

```diff
@@ -8,6 +8,7 @@ class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     StoreProvider.dispatch(context, GetPokemonsAction());
+    StoreProvider.dispatch(context, GetPokemonDetailsAction(pokemonName: 'ditto'));

```

Comment:
kindly place the ditto string in a constant and have a todo comment as well 🙂

Reply:
resolved